static/js/components/email/EmailCompositionDialog.js

```diff
@@ -46,13 +47,29 @@ export default class EmailCompositionDialog extends React.Component {
     if (!this.props.activeEmail) return null;
 
     const {
-      activeEmail: { fetchStatus, inputs },
+      activeEmail: { fetchStatus, inputs, supportsAutomaticEmails },
       title,
       dialogVisibility,
       closeAndClearEmailComposer,
       closeEmailComposerAndSend,
       updateEmailFieldEdit
     } = this.props;
+    let automaticCheckbox;
+    if (supportsAutomaticEmails) {
+      automaticCheckbox = <Checkbox
+        label="Automatically send this message in the future, whenever new users join who meet these criteria"
+        className="email-automatic"
+        checked={inputs.sendAutomaticEmails || false}
+        value={true}
+        onCheck={e => {
+          updateEmailFieldEdit('sendAutomaticEmails', {
+            target: {
+              value: e.target.checked
+            }
+          });
+        }}
+      />;
+    }
```

Comment:
could we move this into a helper method? then in the JSX below we could do

```js
{ supportsAutomaticEmails ? this.automaticCheckbox() : null }
```

or something similar.